samples/counter/README.md

```diff
@@ -0,0 +1,27 @@
+Counter
+=======
+
+This is a very basic demo of Redwood.
+
+
+Prerequisites
+-------------
+
+In order to build and run these applications you'll need to have the following installed:
+- gradle
+- git-lfs
+
+
+Running Counter on Web
+----------------------
+
+To simply view the sample app, open: [[https://cashapp.github.io/redwood/latest/counter/]]
```

Comment:
I think this was MediaWiki syntax? Markdown will linkify raw links, or you can use `[this](syntax)` for linking plain text.

```suggestion
To simply view the sample app, open: https://cashapp.github.io/redwood/latest/counter/
```